packages/datadog-instrumentations/src/grpc/server.js

```diff
@@ -158,7 +158,13 @@ if (nodeMajor <= 14) {
   })
 }
 
-addHook({ name: '@grpc/grpc-js', versions: ['>=1.0.3'], file: 'build/src/server.js' }, server => {
+addHook({ name: '@grpc/grpc-js', versions: ['>=1.0.3 <1.10.0'], file: 'build/src/server.js' }, server => {
+  shimmer.wrap(server.Server.prototype, 'register', wrapRegister)
+
+  return server
+})
+
+addHook({ name: '@grpc/grpc-js', versions: ['>=1.10.0'], file: 'build/src/server.js' }, server => {
```

Comment:
We probably shouldn't be considering these versions supported at all until we have tests working to prove it actually works. 🤔

Reply:
It's unclear from the PR description if it's just the tests that are broken or the functionality. In any case, since this is a supported module, we need to make sure that it's working properly for new versions, otherwise tracing will stop working for users when they upgrade the module.

Reply:
> We probably shouldn't be considering these versions supported at all until we have tests working to prove it actually works. 🤔

That's fair. Since it makes package installs behave strangely for whatever reason, I've removed the new boundary anyway.